sw/device/silicon_creator/manuf/BUILD

```diff
@@ -0,0 +1,31 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("//rules:opentitan_test.bzl", "cw310_params", "opentitan_functest", "verilator_params")
+
+opentitan_functest(
+    name = "ft",
+    srcs = ["ft.c"],
+    cw310 = cw310_params(
```

Comment:
do you want to run this test with ROM only? or test ROM too? if ROM only. then you can add:
```
targets = [
    "cw310_rom",
    "verilator",
],
```

Reply:
Choosing `cw310_rom` for now. Will add verilator once we have otp scaffolding.